code/modules/events/rogue_drones.dm

```diff
@@ -10,12 +10,7 @@
 		if(C.name == "carpspawn")
 			possible_spawns.Add(C)
 
-	//25% chance for this to be a false alarm
-	var/num
-	if(prob(25))
-		num = 0
-	else
-		num = rand(2,6)
+	var/num = rand(4,10)
```

Comment:
Yes I'm sure. They can very much rek people if all 4 end up being hostile due to the RNG being a bitch.